src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rfc2898DeriveBytes.cs

```diff
@@ -11,8 +11,27 @@ namespace System.Security.Cryptography
 {
     public class Rfc2898DeriveBytes : DeriveBytes
     {
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA5350", Justification = "HMACSHA1 is needed for compat. (https://github.com/dotnet/corefx/issues/9438)")]
+        private const int MinimumSaltSize = 8;
+        
+        private readonly byte[] _password;
+        private byte[] _salt;
+        private uint _iterations;
+        private HMAC _hmac;
+        private int _blockSize;
+
+        private byte[] _buffer;
+        private uint _block;
+        private int _startIndex;
+        private int _endIndex;
+
+        public HashAlgorithmName HashAlgorithm { get; }
```

Comment:
Do we have a style rule on that?  I've been burned by forgetting auto-props in Clone-type methods before because they weren't up near the fields.  So instead of thinking of auto-props as "properties with automatic fields" I think of them as "fields with automatic properties".